quiz2/app/views/ideas/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<% @ideas.each do |idea| %>
+    <div class="card mb-3 bg-yellow">
+        <div class="card-body">
+            <h5 class="card-title"><%= link_to idea.title, idea_path(idea),class:"pink-title" %></h5>
+            <p class="card-text"><%= idea.description %></p>
+            <p class="card-text">By <%= idea.user.first_name %></p>
+            <p class="card-text">Likes: <%= idea.likes.count %></p>
+            <% @like = idea.likes.find_by(user: current_user) %> 
```

Comment:
`current_user` is not defined in the controller. You need to define it in application controller in which ideas controller inherits from.